modules/consensus/accept_test.go

```diff
@@ -321,13 +321,13 @@ func TestInconsistentCheck(t *testing.T) {
 	// Corrupt the consensus set.
 	var sfod types.SiafundOutputID
 	var sfo types.SiafundOutput
-	for id, output := range cst.cs.siafundOutputs {
+	cst.cs.db.forEachSiafundOutputs(func(id types.SiafundOutputID, output types.SiafundOutput) {
 		sfod = id
 		sfo = output
-		break
-	}
+	})
```

Comment:
does this forEach work?

You are passing a function to the database that sets sfod to id and sfo to output, but then it looks like the values are only changes once?

I guess I don't understand the construction here.